pt/lessons/advanced/error-handling.md

```diff
@@ -82,7 +82,7 @@ end
 
 ## Novos Erros
 
-Enquanto Elixir inclui uma série de tipos de erro nativos como `RuntimeError`, mantemos a capacidade de criar a nossa própria se precisamos de algo específico. Criar um novo erro é fácil com o `defexception/1` macro que aceita convenientemente `:message` opções para definir uma mensagem de erro padrão:
+Enquanto o Elixir inclui uma série de tipos de erro nativos, como `RuntimeError`, nós mantemos a capacidade de criar a nosso próprio se precisamos de algo específico. Criar um novo erro é fácil com o `defexception/1` macro que aceita convenientemente a opção `:message` para definir uma mensagem de erro padrão:
```

Comment:
"... capacidade de criar **o** nosso próprio se...", just a minor typo with the article used here.